jerry-debugger/jerry-client-ws.html

```diff
@@ -2,31 +2,45 @@
 <html>
 <head>
 <meta charset="UTF-8">
+<!-- Latest compiled and minified CSS -->
+<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
```

Comment:
Hmmm, good point. Then there are two possibility, upload the minified boostrap files ( imho bad solution ) or just delete the bootstrap. What do you think?